### PR TITLE
Update rpc-endpoints.md

### PR DESCRIPTION
**Pull request by gaia**
Removed MaticVigil and NOWNodes, both dead.

### CODE REVIEW

File: docs/pos/reference/rpc-endpoints.md [hunk 1]
```diff
@@ -128,7 +127,6 @@ You can sign up for a dedicated free RPC URL at the following:
 * [SettleMint](https://docs.settlemint.com/docs/polygon-connect-to-a-node)
 * [Tatum](https://tatum.io/)
 * [WatchData](https://docs.watchdata.io/blockchain-apis/polygon-api)
-* [NOWNodes](https://nownodes.io/nodes/polygon-matic)
```
Comment (ashucoder9) [inline docs/pos/reference/rpc-endpoints.md:131]: Is this RPC provider really down? I can see their page and options to setup a Polygon node.

**Review by ashucoder9 — COMMENTED** [review]
Left a comment.

**Comment by gaia** [comment]
Apologies, nownodes.io didn't resolve for me and I thought the domain was dead. Turns out it was the Pi-Hole DNS filtering (more specifically, [this list](https://github.com/NethServer/dns-community-blacklist/blob/master/firebog-crypto.dns#L12345))

File: docs/pos/reference/rpc-endpoints.md (hunk 1 above)
Comment (DannyS03) [reply docs/pos/reference/rpc-endpoints.md:131]:
```suggestion
* [NOWNodes](https://nownodes.io/nodes/polygon-matic)
```

**Review by DannyS03 — CHANGES_REQUESTED** [review]

File: docs/pos/reference/rpc-endpoints.md (hunk 1 above)
Comment (DannyS03) [inline docs/pos/reference/rpc-endpoints.md:131]:
```suggestion
* [NOWNodes](https://nownodes.io/nodes/polygon-matic)
```

**Comment by cerberushades** [comment]
Hey @gaia, can you please edit the PR according to Danny's suggestion?

**Comment by gaia** [comment]
all set? I've resolved the review

**Comment by DannyS03** [comment]
We don't need to remove NOWNodes @gaia

**Pull request closed without merging**